app/scripts/controllers/swaps.js

```diff
@@ -73,6 +72,7 @@ const initialState = {
     topAggId: null,
     routeState: '',
     swapsFeatureIsLive: true,
+    useNewSwapsApi: false,
```

Comment:
Instead of `useNewSwapsApi`, should we use a more meaningful setting name and value, like API version number?  `new` is a relative term and isn't very meaningful after a certain amount of time.

Reply:
I think this will only be in place for a few weeks. Once the switch to v2 is made, this property will be removed.

Reply:
I agree with you, @darkwing , if it would be a long term solution. As Dan mentioned, once we switch to v2 and we feel comfortable with it, this and calls to v1 will be removed.